src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -53,6 +57,52 @@ public UserResource(UserService userService, LibraryCardService libraryCardServi
         this.gson = new Gson();
     }
 
+    @GET
+    @Produces("application/json")
+    @Path("/role/{roleName}")
+    @RolesAllowed({ADMIN, SIGNINGOFFICIAL})
+    public Response getUsers(@Auth AuthUser authUser, @PathParam("roleName") String roleName) {
+        try {
+            User user = userService.findUserByEmail(authUser.getName());
+            if (Objects.nonNull(roleName)) {
+                boolean valid = EnumSet.allOf(UserRoles.class)
+                  .stream()
+                  .map(UserRoles::getRoleName)
+                  .map(String::toLowerCase)
+                  .anyMatch(roleName::equalsIgnoreCase);
```

Comment:
We just added this for another PR, so now it's time to think about DRY. This could be a static method on `UserRoles`, i.e. `static boolean UserRoles.isValid(String roleName) {... do the work ...};`